code/renderers/svelte/src/components/PreviewRender.svelte

```diff
@@ -16,6 +20,8 @@
     props = {},
     /** @type {{[string]: () => {}}} Attach svelte event handlers */
     on,
+    /** @type {boolean} whether this level of the decorator chain is the last, ie. the actual story */
+    isOriginalStory,
```

Comment:
But this storyFn is always the complete decorated one gotten from either:
https://github.com/storybookjs/storybook/blob/595a08c44141eeadf4768ae7b54a68cd1e56c304/code/renderers/svelte/src/render.ts#L154-L157
or
https://github.com/storybookjs/storybook/blob/595a08c44141eeadf4768ae7b54a68cd1e56c304/code/renderers/svelte/src/render.ts#L77-L87

So it would always be false I guess?

Reply:
Or maybe this is for the case there are no decorators?

Reply:
Exactly, undecorated.